charts/fluent-operator/values.yaml

```diff
@@ -61,7 +61,7 @@ fluentbit:
   output:
     es:
       enable: false
-      host: "<Elasticsearch url like elasticsearch-logging-data.kubesphere-logging-system.svc>"
+      host: "<Elasticsearch url like elasticsearch-logging-data.fluent.svc>"
```

Comment:
we can keep the es namespace as it is before : elasticsearch-logging-data.kubesphere-logging-system.svc